src/kml/src/main/java/org/geoserver/kml/regionate/ExternalSortRegionatingStrategy.java

```diff
@@ -167,11 +167,11 @@ void buildIndex(Connection conn) throws Exception {
         try (Statement st = conn.createStatement()) {
             st.execute(
                     "CREATE TABLE FEATUREIDX(" //
```

Comment:
They were "create cached table" at first but then I reverted it back so that I can examine the database content easily to fix testBigGeometries failure. Putting it back on. Thank you!

Reply:
done